src/internal/testutil/identity.go

```diff
@@ -143,10 +157,16 @@ func GetOIDCTokenForTrustedApp(t testing.TB, testClient *client.APIClient) strin
 		return http.ErrUseLastResponse
 	}
 
+	redirectPort := "1657"
+
+	if realEnv {
+		redirectPort = strconv.Itoa(int(testClient.GetAddress().Port + 7))
```

Comment:
Because we want to support running multiple integration tests concurrently, we need to configure the redirect port based on the dynamically generated port from mockPachd. Otherwise, we'll have collisions if multiple instances tried to claim port 1657.

The code that sets them is in real_env.go:

```
identityAddr := realEnv.PachClient.GetAddress().Host + ":" + strconv.Itoa(int(realEnv.PachClient.GetAddress().Port+8))
```